src/pages/home/ReportScreen.tsx

```diff
@@ -372,7 +372,7 @@ function ReportScreen({route, navigation}: ReportScreenProps) {
 
     if (isTransactionThreadView) {
         const transactionID = getTransactionIDFromReportAction(parentReportAction);
-        const transactionViolations = transactionID && allReportViolations ? (allReportViolations as Record<string, OnyxTypes.TransactionViolations>)[transactionID] : undefined;
+        const transactionViolations = transactionID && allReportViolations ? allReportViolations[transactionID] : undefined;
```

Comment:
This fixed https://github.com/Expensify/App/actions/runs/16919414682/job/47941472731 which was introduced in https://github.com/Expensify/App/pull/67065.

cc @roryabraham I think that's why you got pinged in https://expensify.slack.com/archives/C01GTK53T8Q/p1755023194590859